README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=CDCgov_prime-data-input-client&metric=alert_status)](https://sonarcloud.io/dashboard?id=CDCgov_prime-data-input-client) [![Sonar coverage](https://shields.io/sonar/coverage/CDCgov_prime-data-input-client?server=https://sonarcloud.io)](https://sonarcloud.io/dashboard?id=CDCgov_prime-data-input-client)
 
 ## Table of Contents
+  - [Overview](#overview)
+  - [Getting Started](#getting-started)
+  - [Contributing](#contributing)
 
-  - [Developing locally](#developing-locally)
-  - [Running outside of docker](#running-outside-of-docker)
-  - [Support admin](#support-admin)
-  - [Roles](#roles)
-  - [Database cleaning](#database-cleaning)
-  - [Rollbacks](#rollbacks)
-  - [API testing](#api-testing)
-  - [Tests](#tests)
-  - [E2E tests](#e2e-tests)
-  - [Twilio](#twilio)
-  - [MailHog](#mailhog)
-  - [Frontend](#frontend)
-    - [Linters](#linters)
-    - [Storybook and Chromatic](#storybook-and-chromatic)
-  - [PR Conventions](#pr-conventions)
-  - [Cloud Environments](#cloud-environments)
-  - [Cloud Architecture](#cloud-architecture)
-  - [Deploy](#deploy)
-    - [Revert to a Previous Release](#revert-to-a-previous-release)
-    - [Deploy With Action](#deploy-with-action)
+## Overview
```

Comment:
⚠️ Got the content for the "Overview" section from our product vision doc https://docs.google.com/document/d/1Fk7mu7ECsxiVZ_6W1KJcQ_dBIfo46dTu4GdBrbaZGDY/edit

Specifically the "Vision" and "Mission" sections